### DISCUSSION TITLE
Clean backups using DEFAULT_BACKUP_BEGIN

### DISCUSSION THREAD

**Discussion by lordraiden** [tiredofit/docker-db-backup#313] [Q&A] [answered]
If I understood well the documentation default_cleanup_time only works with default_backup_interval.
But is there a way to program the backup with  default_backup_begin using cron and still being able to clean old backups?

```
- DEFAULT_BACKUP_INTERVAL=5760  # Backup every 5760 minutes 4 days
# - DEFAULT_BACKUP_BEGIN="0 2 * * 1,5"      # “At 02:00 on Monday and Friday.”
- DEFAULT_CLEANUP_TIME=14400    # Cleanup backups after 10 days, only works with backup_interval
```

**Comment by tiredofit** [comment, marked as answer]
Cleanup occurs after every backup so you should be good with using cron...
